pom.xml

```diff
@@ -8,6 +8,7 @@
     <artifactId>jv-jdbc-intro</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+
```

Comment:
Minor, but consider removing that

Reply:
> Minor, but consider removing that

Sorry, it provokes an error. If you don't mind, I'd rather level it there